examples/usart.rs

```diff
@@ -1,25 +1,26 @@
 #![no_main]
 #![no_std]
 
-
 extern crate panic_halt;
 
-
 use lpc8xx_hal::{
-    prelude::*,
-    Peripherals,
-    cortex_m_rt::entry,
-    usart::BaudRate,
+    cortex_m_rt::entry, prelude::*, syscon::clocksource::PeripheralClockConfig, Peripherals,
 };
 
-
 #[entry]
 fn main() -> ! {
     let p = Peripherals::take().unwrap();
 
-    let mut swm    = p.SWM.split();
+    let mut swm = p.SWM.split();
     let mut syscon = p.SYSCON.split();
 
+    // TODO
+    swm.handle = swm
+        .handle
+        .disable(&mut syscon.handle)
+        .enable(&mut syscon.handle);
```

Comment:
I'm fine with merging it like this, given the comment is updated (suggestion above) and we open a separate issue to track this problem.